### ISSUE TITLE
Add sections for manual instrumentation in each language of instrumentation docs

### ISSUE THREAD

**Issue by pranay01** [SigNoz/signoz.io#178]
Many users also want to send manual traces to SigNoz. This should be more clearly explained in the docs.

**Comment by srikanthccv** [comment]
@ankit01-oss @Calm-Rock consider prioritizing adding some manual instrumentation instructions also. Many users have started looking at those instructions in our docs and often we refer them to git commits instead of docs.

**Comment by makeavish** [comment]
Add docs on how to instrument functions with decorators in Python and Java.
Also in java there's java agent config to instrument a specific methods: `OTEL_INSTRUMENTATION_METHODS_INCLUDE`

**Comment by ankit01-oss** [comment]
Yes, this is being taken up by a freelancer, most probably next week.

@Calm-Rock for these docs, we can get Anukul to share these in a notion doc. I am guessing these docs will need to get reviews from individual devs, and notion might be better.
